test/v2/models/responses/hateoas/HateoasDepartureMessageIdsResponseSpec.scala

```diff
@@ -41,25 +47,36 @@ class HateoasDepartureMessageIdsResponseSpec extends AnyFreeSpec with Matchers w
         .getOrElse("not set")
 
       s"with a valid message response and receivedSince $set, create a valid HateoasDepartureMessageResponse" in {
-        val messageIds = (for {
-          m1 <- arbitrary[MessageId]
-          m2 <- arbitrary[MessageId]
-          m3 <- arbitrary[MessageId]
-        } yield Seq(m1, m2, m3)).sample.value
+
         val departureId = arbitrary[DepartureId].sample.value
-        val actual      = HateoasDepartureMessageIdsResponse(departureId, messageIds, dateTime)
+        val responses = (for {
+          messageId1 <- arbitrary[MessageId]
+          messageId2 <- arbitrary[MessageId]
+          messageId3 <- arbitrary[MessageId]
+        } yield Seq(
+          genMessageSummary(Some(messageId1)).arbitrary.sample.value,
+          genMessageSummary(Some(messageId2)).arbitrary.sample.value,
+          genMessageSummary(Some(messageId3)).arbitrary.sample.value
+        )).sample.value
```

Comment:
As mentioned elsewhere, you can just use an arbitrary[MessageSummary] call rather than this yielded sequence